config/params.php

```diff
@@ -62,7 +61,6 @@
             LoggerInterface::class => [LoggerInterfaceProxy::class, LogCollector::class],
             EventDispatcherInterface::class => [EventDispatcherInterfaceProxy::class, EventCollector::class],
             ClientInterface::class => [HttpClientInterfaceProxy::class, HttpClientCollector::class],
-            CacheInterface::class,
```

Comment:
Why?

Reply:
Seems, code don't work with integer keys.

Reply:
CacheInterface should be also proxied. Could you add it then?

Reply:
Hm, proxy and collector for `CacheInterface` not exist.